crates/ruff/tests/integration_test.rs

```diff
@@ -1523,7 +1581,17 @@ fn check_hints_hidden_unsafe_fixes() {
     exit_code: 1
     ----- stdout -----
     -:1:1: RUF901 [*] Hey this is a stable test rule with a safe fix.
+    Suggested fix:
+      1 |+# fix from stable-test-rule-safe-fix
+
+        Run `ruff check --fix` to apply this fix.
```

Comment:
Looking at this more holistically, I wonder if this message is necessary since we have a call to action at the end:

> 1 fixable with the `--fix` option

Reply:
I saw that as well, but my intuition was that it is helpful to have both the details and the summary.